apps/prairielearn/elements/pl-sketch/pl-sketch.mustache

```diff
@@ -0,0 +1,50 @@
+<div class="pl-sketch" id="pl-sketch-{{uuid}}">
+    <div class="pl-sketch-controls" style="width: {{width}}px">
+        <div class="pl-sketch-control-group">
+            <div class="pl-sketch-control" id="undo"><div class="fa fa-undo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="redo"><div class="fa fa-redo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="pen"><div class="fa fa-pen fa-2x"></div></div>
+            <div class="pl-sketch-control" id="eraser"><div class="fa fa-eraser fa-2x"></div></div>
+        </div>
```

Comment:
The use of `id` here is probably a bad idea. Any other element that uses the same id (including more than one sketch in the same page) would cause an issue here.

Reply:
As for what to do instead: either use a distinct class on each of these and then select based on that, or include the `uuid` in the ID to ensure they're unique on the page.

Reply:
Like with the `id` above, you need to add the `uuid` to the end.

Reply:
Update:
New lines are as follows - 
```
<div class="pl-sketch-control" id="undo-{{uuid}}"><div class="fa fa-undo fa-2x"></div></div>
<div class="pl-sketch-control" id="redo-{{uuid}}"><div class="fa fa-redo fa-2x"></div></div>
<div class="pl-sketch-control" id="pen-{{uuid}}"><div class="fa fa-pen fa-2x"></div></div>
<div class="pl-sketch-control" id="eraser-{{uuid}}"><div class="fa fa-eraser fa-2x"></div></div>
```

And can be referenced in pl-sketch.js as '#pen-' + uuid, '#undo-' + uuid,...